modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -648,6 +650,17 @@ template<typename Net> class Params {
         return *this;
     }
 
+    /** @brief Configures optimization level for ONNX Runtime.
+
+    @param opt_level [optimization level]: Default is 99 (all). Valid values are 0 (disable), 1 (basic), 2 (extended), 99 (all).
```

Comment:
Updated comment. When opt_level  is not configured SetGraphOptimizationLevel  is not being called. It is up to ONNX runtime to set whatever it wishes. Assume it will set 99 (all)